src/pages/services.tsx

```diff
@@ -36,23 +55,59 @@ const ServicesPage = ({ data, location }: ServicePageProps) => {
 
   return (
     <>
-      <Aurora type={AuroraType.Pink} />
-      <Layout transparentHeader={true} leadbox={leadbox}>
-        <SEO
-          title={`${t('services.title')} | Satellytes`}
-          location={location}
-        />
-        <Grid>
-          <GridItem>
-            <PageTitle>{t('services.title')}</PageTitle>
-          </GridItem>
-
-          <GridItem xs={12} md={8}>
-            <LargeText as={'h2'}>{t('services.subheading')}</LargeText>
-            <MarkdownAst htmlAst={data.markdownRemark.htmlAst} />
-          </GridItem>
-        </Grid>
-      </Layout>
+      <SEO title={`${t('services.title')} | Satellytes`} location={location} />
+      <LayoutV2
+        transparentHeader={true}
+        light={true}
+        leadbox={leadbox}
+        hero={
+          <>
+            <Aurora />
+            <HeroContainer>
+              <ServicesPageTitle>{t('services.title')}</ServicesPageTitle>
+            </HeroContainer>
+          </>
+        }
+      >
+        <ContentBlockContainer>
+          <SectionHeader
+            headline={t('services.header.title')}
+            kicker={t('services.header.kicker')}
+          >
+            {t('services.header.text')}
+          </SectionHeader>
+        </ContentBlockContainer>
+        <ContentBlockContainer>
+          <HeaderBlock
+            headline={t('services.contentblocks.first.title')}
+            illustration="rocket_011"
+            large
+            topline={t('services.contentblocks.first.kicker')}
+          >
```

Comment:
I have created the section header and blog header from the  header block. What's missing is this part. It's okay for this PR but my main goal is to solve the question how to provide larger blocks of text & headlines which will include this part for sure.

I see that this special arrangement of headline + illustration and text is only available on services, so a local component could work wonder here before creating some shareable component. You would be very fast to assemble the different components without the story overhead for example.